### PR TITLE
first feedback

### PR DESCRIPTION
**Pull request by towardsnewtech**
1. landing page popup
2. update mobile menu.
3. update roadmap link

### CODE REVIEW

File: components/DesktopMenu/DesktopMenu.tsx [hunk 1]
```diff
@@ -55,6 +55,10 @@ const DesktopMenu = () => {
         </button>
         {isMenuOpen && (
           <div className="absolute right-0 top-[45px] z-200 inline-flex flex-col items-start uppercase justify-between space-y-4 p-4 bg-[black] dark:bg-white shadow-md rounded-lg  font-quicksand text-sm">
+            <div className="text-gray-400 cursor-not-allowed ">Allowlist</div>
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]: Allowlist is missing a link right?

File: components/Enter/EnterPage.tsx [hunk 2]
```diff
@@ -14,6 +16,13 @@ const EnterPage = () => {
   const contentRef = useRef()
   const buttonRef = useRef()
 
+  const currentTime = new Date().getTime()
+  const shipDay = "30 Jun 2023 09:00:00 UTC"
+  const alertEndDay = "2 Jul 2023 09:00:00 UTC"
+
+  const shouldOpenModal =
+    currentTime >= new Date(shipDay).getTime() && new Date(alertEndDay).getTime() >= currentTime
```
Comment (sweetmantech) [inline components/Enter/EnterPage.tsx]: niccceeeee 💯

File: components/RoadmapPage/Stage.tsx [hunk 3]
```diff
@@ -50,8 +50,12 @@ const Stage: FC<StageProps> = ({
   })
 
   useEffect(() => {
-    if (new Date(stageData.date).getTime() - new Date().getTime() > 0 || !stageData.date)
+    if (new Date(stageData.date).getTime() - new Date().getTime() >= 0 || !stageData.date) {
```
Comment (sweetmantech) [inline components/RoadmapPage/Stage.tsx]:
```suggestion
const shouldBeLocked = new Date(stageData.date).getTime() - new Date().getTime() >= 0 || !stageData.date
    if (isUnlocked) {
```

File: components/RoadmapPage/data.json [hunk 4]
```diff
@@ -46,15 +46,15 @@
...
+    "link": "https://cre8ors.com/claim",
     "text": "Burn a Cre8ors Claim Ticket to redeem a Cre8ors Passport on June 30th 2023 via\nour public launch party in partnership with Magic Eden."
   },
   {
     "backImg": "/assets/Roadmap/allowlist.svg",
     "label": "Allowlist quiz",
     "date": "2023-07-01",
     "certain": true,
-    "link": "/",
+    "link": "http://everythingcorp.cre8ors.com/",
```
Comment (sweetmantech) [inline components/RoadmapPage/data.json]:
```suggestion
    "link": "https://everythingcorp.cre8ors.com/",
```

**Review by SweetmanTech — APPROVED** [review]
small feedback 🚢

File: components/Enter/EnterPage.tsx (hunk 2 above)
Comment (sweetmantech) [reply components/Enter/EnterPage.tsx]: actually, this check makes it hard to test if changes are working.

can you remove the `startTime` so I can test now? I will not ship until popup should be active.

keep the `endTime` so it stops popup automatically

File: components/Enter/EnterPage.tsx [hunk 5]
```diff
@@ -14,6 +16,13 @@ const EnterPage = () => {
   const contentRef = useRef()
   const buttonRef = useRef()
 
+  const currentTime = new Date().getTime()
+  const shipDay = "30 Jun 2023 09:00:00 UTC"
```
Comment (sweetmantech) [inline components/Enter/EnterPage.tsx]: remove `shipDay` and only keep `alertEndDay` 

I will manually ship on Friday 🚢

**Review by SweetmanTech — COMMENTED** [review]
remove `shipDay` so we can test before rollout

File: components/RoadmapPage/Stage.tsx [hunk 6]
```diff
@@ -49,10 +49,12 @@ const Stage: FC<StageProps> = ({
...
 
+  const shouldBeLocked =
+    new Date(stageData.date).getTime() - new Date().getTime() >= 0 || !stageData.date
+
   useEffect(() => {
-    if (new Date(stageData.date).getTime() - new Date().getTime() > 0 || !stageData.date)
-      setIsLocked(true)
-  }, [stageData])
+    setIsLocked(shouldBeLocked)
+  }, [shouldBeLocked])
```
Comment (sweetmantech) [inline components/RoadmapPage/Stage.tsx]: does this need to be stored in `useState`?

why note just directly access the `shouldBeLocked` variable and remove the `useEffect` hook and the `isLocked` state variable? 🚮

**Review by SweetmanTech — APPROVED** [review]
one small comment 📝

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**